src/nav.js

```diff
@@ -0,0 +1,51 @@
+import homeFunc from './home';
+import createContact from './contact';
+import createMenu from './menu';
+
+// const container = document.querySelector('.contain');
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.